### ISSUE TITLE
Font kerning with PDF created with Microsoft Office 2013

### ISSUE THREAD

**Issue by orschiro** [bohoomil/fontconfig-ultimate#68]
Hi @bohoomil ,

I encounter a font kerning issue with a PDF that has been created using Microsoft Office 2013. I use `Evince` to display the PDF which looks as follows:

![screenshot](https://cloud.githubusercontent.com/assets/170947/5665374/41fc2d8c-9758-11e4-94e1-27de3b243d2d.png)

Information about the PDF ([download](https://transfer.sh/uOMLl/transition-training-day-2015-programme.pdf)):

```
pdfinfo transition\ training\ day\ 2015_Programme.pdf
Author:         Marijke de Pous
Creator:        Microsoft® Word 2013
Producer:       Microsoft® Word 2013
CreationDate:   Thu Jan  8 11:16:22 2015
ModDate:        Thu Jan  8 11:16:22 2015
Tagged:         yes
UserProperties: no
Suspects:       no
Form:           none
JavaScript:     no
Pages:          1
Encrypted:      no
Page size:      594.96 x 842.04 pts (A4)
Page rot:       0
File size:      84394 bytes
Optimized:      no
PDF version:    1.5
```

Used version of fontconfig-ultimate:
```
~ ❯ pkg-grep infina
cairo-infinality-ultimate 1.14.0-2
fontconfig-infinality-ultimate 2.11.1-24
freetype2-infinality-ultimate 2.5.5-1
lib32-cairo-infinality-ultimate 1.14.0-2
lib32-fontconfig-infinality-ultimate 2.11.1-3
lib32-freetype2-infinality-ultimate 2.5.5-1
```

Can you confirm this issue?

Thankfully,

Robert

**Comment by bohoomil** [comment]
Yes, but this has nothing to do with Infinality, `freetype2` and the like: there are neither embedded fonts in the document, nor any other fonts (in ttf/otf/t1/… format) that `pdffonts` could identify (you could use `Evince -> Properties… -> Fonts` to check this out). It can't be even converted to an editable text document (in any format, be it `doc`, `rtf`, `txt`). The content must have been written as an image with anti-aliasing information Evince can't read (Zathura can, though). All in all: not a font related issue.

**Comment by orschiro** [comment]
>  there are neither embedded fonts in the document, nor any other fonts (in ttf/otf/t1/… format) that

Thanks for mentioning! I didn't think about that possibility.

> The content must have been written as an image with anti-aliasing information Evince can't read (Zathura can, though)

Tested Zathura and will use it for now whenever Evince fails to show a PDF properly.

>All in all: not a font related issue.

Apologise for bothering with a non-related issue...

Robert
